python/GafferUI/LabelPlugValueWidget.py

```diff
@@ -203,7 +201,7 @@ def __labelDoubleClicked( self, label, event ) :
 		assert( len( self.getPlugs() ) == 1 )
 
```

Comment:
At some point we seem to have lost this :

```
if Gaffer.MetadataAlgo.readOnly( self.getPlug() ) :
     return
```

I don't think that was intentional?

Reply:
No, reinstated [here](https://github.com/GafferHQ/gaffer/pull/3980/commits/f63143d813808c3e74a4491aa3b53782bb45f376) thanks :)